externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/dialect/JdbcDialect.scala

```diff
@@ -85,7 +86,9 @@ abstract class JdbcDialect extends SupportServiceLoader with Logging {
 
 object JdbcDialects extends Logging {
 
+  var defaultFetchSize: Int = 0
   def get(conf: KyuubiConf): JdbcDialect = {
+    defaultFetchSize = conf.get(ENGINE_JDBC_FETCH_SIZE)
```

Comment:
This fetch size may be modified by the conf of different sessions. Can the fetch size be obtained in JdbcOperationManager?

https://github.com/apache/kyuubi/blob/e8e981d4abe8e3be66ac8674f5284a202388a357/externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/operation/JdbcOperationManager.scala#L44-L46